.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,45 @@
+name: Deploy documentation
+
+on:
+    workflow_dispatch:
+    release:
+      types:
+        - published
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout nplinker
+      uses: actions/checkout@v4
+      with:
+        ref: dev
+        fetch-tags: true
+
+    - name: Fetch gh-pages
+      run: git fetch origin gh-pages:gh-pages
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.10'
+
+    - name: Install dependencies
+      run: python3 -m pip install .[dev]
+
+    - name: Sync webapp README
```

Comment:
webapp readme is synced to this repo. But for manual publishing, we are not syncing it.
Either we don't sync it at all, or we sync it for both cases. Both are fine to me.

Reply:
Now it's always sync